pkg/cmd/roachtest/tests/perturbation/framework.go

```diff
@@ -297,6 +297,9 @@ func RegisterTests(r registry.Registry) {
 
 func (v variations) makeClusterSpec() spec.ClusterSpec {
 	opts := append(v.specOptions, spec.CPU(v.vcpu), spec.SSD(v.disks), spec.Mem(v.mem), spec.TerminateOnMigration())
+	if v.perturbationName() == "slowDisk" {
+		opts = append(opts, spec.ReuseNone())
```

Comment:
I'm assuming this is to avoid potential side-effects of cgroups? It's not entirely obvious why `ReuseNone` only applies in this case.